logging/logadmin/logadmin.go

```diff
@@ -311,21 +310,26 @@ func (it *EntryIterator) fetch(pageSize int, pageToken string) (string, error) {
 var slashUnescaper = strings.NewReplacer("%2F", "/", "%2f", "/")
 
 func fromLogEntry(le *logpb.LogEntry) (*logging.Entry, error) {
-	time, err := ptypes.Timestamp(le.Timestamp)
-	if err != nil {
+	if err := le.GetTimestamp().CheckValid(); err != nil {
 		return nil, err
 	}
+	time := le.GetTimestamp().AsTime()
 	var payload interface{}
 	switch x := le.Payload.(type) {
 	case *logpb.LogEntry_TextPayload:
 		payload = x.TextPayload
 
 	case *logpb.LogEntry_ProtoPayload:
-		var d ptypes.DynamicAny
-		if err := ptypes.UnmarshalAny(x.ProtoPayload, &d); err != nil {
+		// I don't know
+		// var d ptypes.DynamicAny
+		// if err := ptypes.UnmarshalAny(x.ProtoPayload, &d); err != nil {
+		// 	return nil, fmt.Errorf("logging: unmarshalling proto payload: %w", err)
+		// }
```

Comment:
Does this need addressing still? Or just needs to be cleaned up?

Reply:
oops, forgot to push this lol.